src/nanotron/models/llama.py

```diff
@@ -594,6 +601,7 @@ def __init__(
     ):
         super().__init__()
         self.input_layernorm = TritonRMSNorm(config.hidden_size, eps=config.rms_norm_eps)
+        self.input_layernorm.linear_type = WeightType.INPUT_WEIGHTS
```

Comment:
not a big fan of adding those here in the modeling. We can either regroup them at the end of this init with a banner comment that says `# For mutransfer` 
Or do them outside the init, maybe in a mutransfer's trainer